src/packages/@ncigdc/utils/trace.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-sequences */
+
+import React from 'react';
+import { branch, renderComponent } from 'recompose';
+
+/**
+* Utility HOC designed to easily log received props.
+
+* usage:
+
+  trace('Trying to fix issue #12')(Button)
+
+  compose(
+    global.trace() // string argument is optional.
+  )
+*/
+
+export default (additionalMessage = '') => Component =>
+  branch(
+    () => process.env.NODE_ENV === 'production',
+    renderComponent(Component),
+  )(
+    props => (
+      additionalMessage && console.log(additionalMessage),
```

Comment:
Could this all be in the console.info? 
something like 
```
console.info(
    `${additionalMessage ? `${additionalMessage}\n` : ''}%ctracing ${Component .displayName} props:\n${'tracing '.replace(
      /./g,
      '=',
    )}${Component .displayName.replace(/./g, '=')}=${'props:'.replace(
      /./g,
      '=',
    )}\n`,
    'color: rgb(22, 138, 96);font-weight: bold;',
    props
  )
```
It would make it easier to filter the logs.